File: Iniciante/1007 - Diferença/bee1007.java
// Beecrowd 1007 - Diferença
import java.io.IOException;
import java.util.Scanner;

public class bee1007 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int A = scan.nextInt();
        int B = scan.nextInt();
        int C = scan.nextInt();
        int D = scan.nextInt();

        int ans = (A * B - C * D);
        System.out.println("DIFERENCA = " + ans);
    }
}

File: Iniciante/1007 - Diferença/bee1007.jl
# Beecrowd 1007 - Diferença

function main()
    A = parse(Float64, readline())
    B = parse(Float64, readline())
    C = parse(Float64, readline())
    D = parse(Float64, readline())

    DIFF = (A*B) - (C*D)
    println("DIFERENCA = ", trunc(Int, DIFF))

end

main()
